### ISSUE TITLE
Suggestions by lucumo (vndb.org)

### ISSUE THREAD

**Issue by micah686** [micah686/VnManager#43]
1) enable white color scheme
2) list view for games instead of grid view
3)adjust the layout of some elements
4)Ability to hide tags/traits

**Comment by Omar-AbdulAzeez** [comment]
I see that you've adapted the game screen from steam but I agree with lucumo on this one... steam has the list of games on the left which takes space so putting a poster of the selected game on the left will take too much space so they put a faded banner in the background of the text instead... both of these approaches are great... the main issues with your current layout is
1. (pointed out by lucumo already) the poster is on the right with the language being left to right
2. using sliders for tags... while it's not a bad idea for something with few tags... if the number is high, navigating through tags is almost impossible. also inconsistencies and incorrect grouping... I only tested one game for now so this may be consistent but still... student club president heroine is a child of character, setting and theme. yet it only appears in one of those groups and it happened to be setting in the game I checked...
3. in the screenshots tab, the screenshot navigation section is on top of the image box. resulting in cutting the lower section of the screenshot
4. vndb and wikipedia links not having a unique or mouse hover look which makes them look like plain text

for the tags I suggest adapting the way vndb does it because it allows you take a look at everything... starting from the big text tags to the small text tags with fixed spacing in a wrap-around dynamic-size text box... I like the idea of separating tags in parent-childs groups tho so you can do that in the same way vndb categorizes tags in 3 major sections (content, sexual content, technical) maybe you can find a way to implement those alongside those tag buttons
I also like the idea of maximum sexual level to blur images but separate this from tags (also violence if that affects it) because there is no reason to filter sexual text... even the argument of parental control fails here because if the game has sexual tags it should just be hidden as a whole... instead include another toggle in the settings whether sexual content tags are on or off by default simply because people either care or don't care about those tags...
and there is always reason to allow the option of going against the default settings (sexual content and spoilers) inside of each entry (game)... if I want to check the spoilers of a game with the current system I have to go into the settings and change default settings which defeats the purpose of a "default" setting
so basically adapting the exact way nvdb handles tags is great

as for the screenshot issue it's as simple as anchoring the bottom side of the image box to the top side of the screenshot navigation section while setting the image box to "fit" mode and fixing the height of the screenshot navigation section
so now no matter the size of the whole program window the bottom section is anchored to the program's bottom side with the image box taking whatever space is left and displays the screenshot in it

for the layout as a whole... put the poster on the left and move the relations section above the summary (and hopefully incorporate links into it as well)
also there is the issue of going into settings and out doesn't put you back in a game's page if you opened it from there
